testdata/configtest.json

```diff
@@ -1850,7 +1850,7 @@
         "credentialsValidator": {
           "requiresKey": true,
           "requiresSecret": true,
-          "requiresBase64DecodeSecret": true
+          "requiresBase64DecodeSecret": false
```

Comment:
Why has this been changed?

Reply:
Also Change Rot.
I'd been struggling to not have AuthenticatedWebsocketApiSupport re-disabled on config loading.
So I'd been using a Key/Secret and setting it to true in configtest.json.
That had (obvious) ramifications, so I ended up deciding that all mock tests should get to auth anyway.

Fixed 029769f1